### ISSUE TITLE
Implementation: Reorganize receipts and disbursements landing pages for candidates in H4CC

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2173]
Candidates and others need to find the ton of helpful information in the receipts and disbursements buckets so we need to reorganize that information to make it easier for them and to house and create new pages we've identified as gaps in the information. Creation of the new pages will be handled in implementation issues at #2111 #2124 #2125 . 

This ticket is to do the necessary re-structuring of the landing pages and update any existing pages as needed.

Proposed new reorganization was done in #2101 by @llienfec and me for content team review.

Once comments are in, it's time to implement:

**To do:**
- [x] Finish collecting edits from content team
- [x] Research how to make "new" pages but keep old slugs
- [x] Make edits and decisions based on comments
- [x] Update pages as needed. 
- [x] Add any needed redirects after new, revised or moved pages are published

**Comment by llienfec** [comment]
Sent restructuring document to content team again for one final review.  Hope to move forward with restructuring next week.

**Comment by llienfec** [comment]
We have comments back from content team and are figuring out how to move forward making these changes.

### Research task:
**Goal:** Want the new page to have the same URL as the old page.  BUT we need to change template.  

**Research questions:** 
1. Can we change a page's template in wagtail without changing the slug?
1. If not, can we find a way to "swap" the old page for the new, deleting the old?

**Method:**
- Test some of the methods using dev or feature.  Ask engineers which is best.

**Comment by llienfec** [comment]
I tested replacing pages in Wagtail.  Results weren't good.

### Findings:
- We can't update a page's template (which would have kept all links the same)
- We can't create a new page and replace it b/c the slugs differ and it breaks internal & external links
- We can't unpublish the old page and swap the slugs b/c even though the slugs are the same, it breaks internal links.

### Solution:
- Create new pages, with slightly different slugs
- Create redirects from the old pages to the new pages (preserving both internal & external links)
- Unpublish/delete the old pages?

**Comment by llienfec** [comment]
Talked to @johnnyporkchops just to check that this seemed like the right approach.  He didn't have additional info about template swapping and said that this makes sense.  So barring any big discovery, @dorothyyeager and I are ready to go forward with the solution above.

**Comment by AmyKort** [comment]
Thanks!

**Comment by dorothyyeager** [comment]
Unfortunately, we  just discovered a bug with the resource page template that we will need to fix before we publish any new pages in H4CC that use that template.

**Comment by llienfec** [comment]
- Went through and added citations to these new pages (used info from old pages + an AO cite 1993-12).
- Updated candidate `taking in receipts` page to follow our updated format and added links to draft pages.
- Updated candidate `making disbursements` to remove info that now belongs in the receipts section.

**Comment by dorothyyeager** [comment]
Above referenced bug turned out not to be a bug after all and we are not blocked.

Tweaks have been made to the making disbursements section to add links to new forthcoming pages on joint fundraising and re-label the "advertising" page as "Advertising and disclaimers" (since 99% of it is about disclaimers).

Tweaked the candidate taking in receipts page to add a link to forthcoming new pages on personal funds and to relabel link to corporate support page.

**Comment by dorothyyeager** [comment]
The new reorganized landing pages are published and all of the pages linked on them have been published as well.
"Taking in receipts" now includes a subsection for "outside spending". Three of the pages on contributions were reorganized and expanded using the "resource page template." We also removed some duplicative content from the page on corporate/labor communications
https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/

"Making disbursements" - we added a new page on joint fundraising, moved coordinated communications to the "outside spending" part of receipts, and re-labeled some other pages.

https://www.fec.gov/help-candidates-and-committees/making-disbursements/

With the reorganization complete for now, we can close this issue.
